### ISSUE TITLE
`corber open ios` is unable to open projects with a space in the name

### ISSUE THREAD

**Issue by ddoria921** [isleofcode/corber#497]
### Environment
Mac OS 10.13.4

### Reproduction
Create a project where the name in the config.xml file includes a space like "My App". 
Then run `corber open ios` 

### Possible Solution
Looking at the source code I see that the `open-app.js` cordova task opens the repo by simply taking the name from the config.xml file and opening the xcode project by that name in the ios directory.

A possible fix to this would be replacing the spaces in the name with `\ ` (a space character following the backslash). This is the way you open files with spaces in the name on a Mac. Not sure if the solution would be the same for Windows.

**Comment by ddoria921** [comment]
Diving into the code there can be a check here to add the necessary backslash https://github.com/isleofcode/corber/blob/master/lib/utils/open-app-command.js#L11

But I noticed there was a comment above pointing to the original file here https://github.com/pwnall/node-open/blob/0c3ad272bfbc163cce8806e64630c623a9cfd8f4/lib/open.js

Looks like the original file had an `escape()` function that handled the backslash scenario. Any reason why is was removed? If it was a mistake I don't mind opening a PR to add it back.
